clouds/snowflake/modules/sql/utils/_CHECK_TABLE.sql

```diff
@@ -0,0 +1,23 @@
+---------------------------
+-- Copyright (C) 2024 CARTO
+---------------------------
+
```

Comment:
1. Rename this to `_VALIDATE_IDENTIFIER` or similar. By the current name I would assume it actually checks a real table exists or doesn't exist or something. 

2. It would be good to have a test for this for consistency with the other utils. My suggestion would be two describe.each . One for VALID identifiers. Another for invalid identifiers. But even one simple happy-path test would suffice.